idl/cherami/cherami.thrift

```diff
@@ -550,7 +550,8 @@ struct ReconfigureInfo {
 
 enum InputHostCommandType {
   ACK,
-  RECONFIGURE
+  RECONFIGURE,
+  DRAIN // to trigger draining of a all connections to this inputhost
```

Comment:
all connections or just the connection on which the reconfigure command was sent ? AFAIK, these commands are sent on a single connection stream.